test/index.js

```diff
@@ -54,18 +50,11 @@ describe('Vero', function(){
   describe('mapper', function(){
     describe('identify', function(){
       it('should map basic identify', function(){
-        test.set({ authToken: 'some-auth-token' });
+        test.set({ authToken: settings.authToken });
```

Comment:
the fixtures are shared between the mapper tests and the functional tests. we could remove it from the fixtures and then exclude it from all the equals. i'm ok with either solution